airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java

```diff
@@ -409,7 +409,7 @@ public void importIntoWorkspace(final AirbyteVersion targetVersion, final UUID w
               // make sure connector definition exists
               try {
                 final StandardSourceDefinition sourceDefinition =
-                    configRepository.getStandardSourceDefinition(sourceConnection.getSourceDefinitionId());
+                    configRepository.getStandardSourceDefinition(sourceConnection.getSourceDefinitionId(), false);
```

Comment:
This would throw an exception instead of being null. Maybe this should return the actual definition and then check if it's tombstoned or not.

Reply:
Good catch, fixed!